test/validation/api/packet/packet.c

```diff
@@ -339,23 +340,37 @@ static void packet_set_inflags(odp_packet_t pkt, int val)
 	odp_packet_has_eth_mcast_set(pkt, val);
 	odp_packet_has_jumbo_set(pkt, val);
 	odp_packet_has_vlan_set(pkt, val);
-	odp_packet_has_vlan_qinq_set(pkt, val);
-	odp_packet_has_arp_set(pkt, val);
 	odp_packet_has_ipv4_set(pkt, val);
-	odp_packet_has_ipv6_set(pkt, val);
 	odp_packet_has_ip_bcast_set(pkt, val);
-	odp_packet_has_ip_mcast_set(pkt, val);
 	odp_packet_has_ipfrag_set(pkt, val);
 	odp_packet_has_ipopt_set(pkt, val);
 	odp_packet_has_ipsec_set(pkt, val);
 	odp_packet_has_udp_set(pkt, val);
-	odp_packet_has_tcp_set(pkt, val);
-	odp_packet_has_sctp_set(pkt, val);
-	odp_packet_has_icmp_set(pkt, val);
```

Comment:
With this change, we're not setting these flags to 1 anywhere in validation. Should we?

Reply:
They are still tested in `packet_test_in_flags()`with some macro trickery.

Reply:
In that case this should be fine.